diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index fdb6429352f284db921bad2d24f7013f674f885d..43de44f2394bba10841197ab13b41d569442e7b8 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -1328,6 +1328,7 @@ if (!is_android && !is_fuchsia) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
