diff --git a/chrome/browser/ui/views/frame/browser_view.cc b/chrome/browser/ui/views/frame/browser_view.cc
index c7df34e9219b8592021f128a780490e03addc390..7bae583d69b3e3f9ac5ffed5dd4e535069e45c8f 100644
--- a/chrome/browser/ui/views/frame/browser_view.cc
+++ b/chrome/browser/ui/views/frame/browser_view.cc
@@ -261,7 +261,7 @@
 #include "ui/views/win/scoped_fullscreen_visibility.h"
 
 // To avoid conflicts with the macro from the Windows SDK...
-#undef LoadAccelerators
+//#undef LoadAccelerators
 #endif
 
 #if BUILDFLAG(ENABLE_ONE_CLICK_SIGNIN)
