diff --git a/components/sync/driver/BUILD.gn b/components/sync/driver/BUILD.gn
index 60b8b3018ee9f2ef059be10eb4ae8bfc2cb56e94..88117597f0a1344da9022dd7cc372e21f3f40db7 100644
--- a/components/sync/driver/BUILD.gn
+++ b/components/sync/driver/BUILD.gn
@@ -110,6 +110,7 @@ static_library("driver") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_sync_driver_sources deps += brave_components_sync_driver_deps
 }
 
 static_library("test_support") {
