diff --git a/components/permissions/permission_request.cc b/components/permissions/permission_request.cc
index 015f54c7108fb773f2bee2f954f2a02187e51781..b70956874be881b554da0bf91bf51e611a7528ad 100644
--- a/components/permissions/permission_request.cc
+++ b/components/permissions/permission_request.cc
@@ -93,6 +93,7 @@ std::u16string PermissionRequest::GetDialogMessageText() const {
     case RequestType::kVrSession:
       message_id = IDS_VR_INFOBAR_TEXT;
       break;
+    BRAVE_PERMISSION_REQUEST_GET_MESSAGE_TEXT_FRAGMENT
   }
   DCHECK_NE(0, message_id);
   return l10n_util::GetStringFUTF16(
@@ -209,6 +210,7 @@ std::u16string PermissionRequest::GetMessageTextFragment() const {
     case RequestType::kVrSession:
       message_id = IDS_VR_PERMISSION_FRAGMENT;
       break;
+    BRAVE_PERMISSION_REQUEST_GET_MESSAGE_TEXT_FRAGMENT
     case RequestType::kWindowPlacement:
       message_id = IDS_WINDOW_PLACEMENT_PERMISSION_FRAGMENT;
       break;
