diff --git a/content/public/browser/web_contents_receiver_set.h b/content/public/browser/web_contents_receiver_set.h
index d1a51c73344de15b40d463ce88360de6a8fb968e..ca2830fa2074bcc4c5c6a63a077e18d4d64c9b53 100644
--- a/content/public/browser/web_contents_receiver_set.h
+++ b/content/public/browser/web_contents_receiver_set.h
@@ -46,6 +46,11 @@ namespace subresource_redirect {
 class SubresourceRedirectObserver;
 }
 
+class BraveDrmTabHelper;
+namespace brave_shields {
+class BraveShieldsWebContentsObserver;
+}
+
 namespace content {
 
 class RenderFrameHost;
@@ -124,6 +129,8 @@ class WebContentsFrameReceiverSetPassKey {
   friend class ::subresource_redirect::SubresourceRedirectObserver;
   friend class ScreenOrientationProvider;
   friend class TestFrameInterfaceBinder;
+  friend class ::BraveDrmTabHelper;
+  friend class ::brave_shields::BraveShieldsWebContentsObserver;
   FRIEND_TEST_ALL_PREFIXES(WebContentsReceiverSetBrowserTest,
                            OverrideForTesting);
 };
