diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
index 016d86d6b6ca590e76138f47d2660999f4bab78c..0ddd3796145176c10fd8b4711fc22b8a206b6c3c 100644
--- a/ios/chrome/app/BUILD.gn
+++ b/ios/chrome/app/BUILD.gn
@@ -167,7 +167,7 @@ if (ios_enable_credential_provider_extension) {
       "//components/keyed_service/core",
       "//ios/chrome/app/application_delegate:app_state_header",
       "//ios/chrome/app/application_delegate:observing_app_state_agent",
-      "//ios/chrome/browser/credential_provider",
+    # "//ios/chrome/browser/credential_provider",
       "//ios/chrome/browser/passwords:store_factory",
       "//ios/chrome/browser/ui/main:browser_interface_provider",
       "//ios/chrome/browser/ui/main:scene_state_header",
