diff --git a/services/network/restricted_cookie_manager.cc b/services/network/restricted_cookie_manager.cc
index 00c4cceb89bd2819cba5c988497aecf6763fc3b9..852292e82eff70b6ef6f8980ab8b2ce65fe0895f 100644
--- a/services/network/restricted_cookie_manager.cc
+++ b/services/network/restricted_cookie_manager.cc
@@ -301,6 +301,7 @@ void RestrictedCookieManager::GetAllForUrl(
   //                                 removing deprecation warnings.
   net_options.set_return_excluded_cookies();
 
+  BRAVE_GETALLFORURL
   cookie_store_->GetCookieListWithOptionsAsync(
       url, net_options,
       base::BindOnce(&RestrictedCookieManager::CookieListToGetAllForUrlCallback,
@@ -461,6 +462,7 @@ void RestrictedCookieManager::SetCanonicalCookie(
       role_, url, site_for_cookies, isolation_info_, cookie_settings(),
       cookie_store_->cookie_access_delegate());
 
+  BRAVE_SETCANONICALCOOKIE
   cookie_store_->SetCanonicalCookieAsync(
       std::move(sanitized_cookie), origin_url, options,
       base::BindOnce(&RestrictedCookieManager::SetCanonicalCookieResult,
